dotcom-rendering/src/web/components/Callout/CalloutComponents.tsx

```diff
@@ -0,0 +1,164 @@
+import { css } from '@emotion/react';
+import {
+	body,
+	neutral,
+	palette,
+	space,
+	textSans,
+} from '@guardian/source-foundations';
+import { Button, SvgShareCallout } from '@guardian/source-react-components';
+import { useState } from 'react';
+import { sanitiseHTML } from '../../../../src/model/sanitise';
+import { decidePalette } from '../../lib/decidePalette';
+
+const descriptionStyles = (format: ArticleFormat) =>
+	css`
+		a {
+			color: ${decidePalette(format).text.richLink};
+			border-bottom: 1px solid ${decidePalette(format).text.richLink};
+			text-decoration: none;
+		}
+		padding-bottom: ${space[4]}px;
+		${body.medium()}
+
+		p {
+			margin-bottom: ${space[3]}px;
+		}
+	`;
+
+export const CalloutDescription = ({
+	description,
+	format,
+}: {
+	description: string;
+	format: ArticleFormat;
+}) => (
+	<div css={descriptionStyles(format)}>
+		<div
+			dangerouslySetInnerHTML={{ __html: sanitiseHTML(description) }}
+		></div>
+		<div>
+			Please share your story if you are 18 or over, anonymously if you
+			wish. For more information please see our{' '}
+			<a href="https://www.theguardian.com/help/terms-of-service">
+				terms of service
+			</a>{' '}
+			and{' '}
+			<a href="https://www.theguardian.com/help/privacy-policy">
+				privacy policy
```

Comment:
Tiny detail but think you're missing a dot at the end of the sentence.

Reply:
Well observed. Will fix.